### ISSUE TITLE
Where is the access token for unit testing?

### ISSUE THREAD

**Issue by hal256** [amzn/amazon-pay-sdk-php#91]
I want to write a unit test for authorization and billing agreement. Is there any way to get a valid access_token without using a browser?

**Comment by shangamesh** [comment]
No. Access_token can bed fetched only via browser after logging in with buyer account which can be used for unit testing
